src/meson.build

```diff
@@ -39,6 +39,11 @@ subdir('libxmlb')
 
 mapfile = 'libxmlb.map'
 vflag = '-Wl,--version-script,@0@/@1@'.format(meson.current_source_dir(), mapfile)
+if get_option('zstd')
+    zstd_sources = ['xb-zstd-decompressor.c']
+else
+    zstd_sources = []
```

Comment:
maybe call this extra_sources or something? I could imagine other things there too.

Reply:
Okay. I am changing now.

Reply:
also, maybe define the [] before the `if get_op` and then use += for the xb-zstd-decompressor

Reply:
https://github.com/hughsie/libxmlb/pull/134/commits/9a751e1576b8fa68d8fe3b41a8d81d0341fd6e3f fixed

Reply:
I think if we will add extra features. we can use extra_sources array. 
Also extra_sources used 2 times in file...